app/dash_app.py

```diff
@@ -278,13 +385,49 @@ def plot_coherence(coh_long):
 temporal_view = Graph(
     id='coherence-matrix',
     figure=plot_coherence(_read_coherence(_coherence_csv(INITIAL_TARGET))),
+    # figure=plot_baseline(_read_baseline(_baseline_csv(INITIAL_TARGET)),
+    #                      _read_coherence(_coherence_csv(INITIAL_TARGET))),
```

Comment:
I see the utility of this during development, but I think commented code like this should be stripped when developmentis done.

Reply:
Indeed, good catch. Fixed